### ISSUE TITLE
Add simple web ui

### ISSUE THREAD

**Issue by fnpanic** [vexxhost/migratekit#6]
It would be great to have a webui for migration kit to simplify the usage.

**Comment by mnaser** [comment]
I think down the line, it would be ideal if Migratekit runs as an agent, and then another service talks to it.  There are some sort of networking problems we would have to think about to make sure the case when the two systems cannot talk to each other directly.

**Comment by fnpanic** [comment]
I agree. I general the architecture could look something like this. You deploy an "agent" in the source and destination and source. They should then register to a central system which is reachable just with public HTTPS which makes deployment way easier. Then the central componet can talk to the "agents". Relaying the traffic over the central point could also make it easier.
This sorts out most of the networking problems. Using something like https://github.com/jpillora/chisel has proven to be very effective in the past. Queueing and job management could happen centrally. 

This require some engineering but with the help of claude 3.5 sonnet most of the heavy lifting is only a proper design.
